src/PowerSystem/power_network.jl

```diff
@@ -40,7 +40,7 @@ struct PowerNetwork <: AbstractPowerSystem
     sbus::Vector{Complex{Float64}}
     sload::Vector{Complex{Float64}}
 
-    function PowerNetwork(data::Dict{String, Array}; remove_lines=Int[])
+    function PowerNetwork(data::Dict{String, Array}; remove_lines=Int[], multi_generators=:aggregate)
```

Comment:
@frapac aggregate in this case will be what we have been doing until now. Right?

Reply:
Exactly. We aggregate the generators to get one generator per PV/slack node, at the end. It's mostly a Julia port of your matlab script, I would say.

Reply:
@frapac got it. thanks!!